docs/usage/integration.md

```diff
@@ -0,0 +1,413 @@
+# Integration
+
+TigerBeetle is designed to guard against bugs not only in its [own code](../TIGER_STYLE.md), but
+at the boundaries, in the application code which interfaces with TigerBeetle.
+This is exhibited by the client's API design, which may be [surprising](#retries) when contrasted
+with a more conventional database.
```

Comment:
Are retries really so different from conventional databases?

Reply:
The client doesn't have a retry limit — it will never bubble a network error up to the application, it will just keep trying.